doc/source/ray-core/package-ref.rst

```diff
@@ -332,3 +332,62 @@ The Ray Command Line API
 .. click:: ray.scripts.scripts:enable_usage_stats
    :prog: ray enable-usage-stats
    :show-nested:
+
+.. _ray-core-exceptions:
+
+Exceptions
+----------
+
+.. autoclass:: ray.exceptions.RayError
+   :members:
+
+.. _ray-core-exceptions-ray-task-error:
```

Comment:
Surprisingly, they have never been documented although they are public APIs

Reply:
I also feel some of the exception hierarchy is weird. I feel we didn't go through the same api review process for exceptions.